site/docs/install.md

```diff
@@ -7,9 +7,9 @@ title: Installing Bazel
 
 See the instructions for installing Bazel on:
 
-*   [Ubuntu Linux (16.04, 15.10, and 14.04)](install-ubuntu.md)
-*   [Mac OS X](install-os-x.md)
-*   [Windows (experimental)](install-windows.md)
+*   [Linux](install-ubuntu.md)
```

Comment:
What's the full list of supported platforms and which ones can you only install by compiling from source?